src/xenia/cpu/backend/x64/x64_backend.cc

```diff
@@ -26,10 +26,23 @@
 #include "xenia/cpu/processor.h"
 #include "xenia/cpu/stack_walker.h"
 
-DEFINE_bool(
-    use_haswell_instructions, true,
-    "Uses the AVX2/FMA/etc instructions on Haswell processors when available.",
-    "CPU");
+DEFINE_int32(x64_extension_mask, -1,
+             "Allow the detection and utilization of specific instruction set "
+             "features.\n"
+             "    0 = x86_64 + AVX1\n"
```

Comment:
Has `xb format` been done between `git add -A` and `git commit`? Arguments containing multi-line string literals are usually moved to a separate, indented line rather than aligned by it.